modules/RTC/TraceablePeerConnection.js

```diff
@@ -1224,25 +1222,31 @@ function extractSSRCMap(desc) {
             }
         }
         for (const ssrc of mLine.ssrcs) {
-            if (ssrc.attribute !== 'msid') {
+            if (!this._usesUnifiedPlan && ssrc.attribute !== 'msid') {
```

Comment:
two negated conditions are a bit hard to read here, is it the same as "do it for  every ssrc in unified or only those with msid in plan B" ? how about doing filter on `mLine.ssrcs` only in plan B one line above ? this way there would be no continue and no eslint ignore.

Reply:
Fixed it in this [PR](https://github.com/jitsi/lib-jitsi-meet/pull/1638)